### PR TITLE
License and copyright

### PR DESCRIPTION
**Pull request by will-moore**
See #3.

@jburel I have removed the copyright as `Copyright TBD` didn't look great.
I guess we could get cookiecutter to ask for a copyright author but it feels like it's up to users whether they add a copyright notice to their code and it's not the job of this app to enforce that they do.

Added a LICENSE (copied from omero-figure).
Updated gitignore.

### CODE REVIEW

**Comment by jburel** [comment]
I think a note should be added to the README indicating to add copyright header to the files

**Comment by will-moore** [comment]
@jburel Done at [7b6500e](https://github.com/ome/cookiecutter-omero-webapp/pull/5/commits/7b6500ee611f977e025e8a46e3fe3ca7130a76f1)

File: README.md [hunk 1]
```diff
@@ -34,3 +34,9 @@ $ omero config append omero.web.apps '"webapp_name"'
 Then you can restart your `omero-web` server and see your app at `<your-server>/webapp_name/`.
 
 For more details, please see the README of your newly created app.
+
+# License and Copyright
+
+All projects that depend on the GPL-licensed `omero-web` must also be licensed as GPL.
```
Comment (jburel) [inline README.md]: AGPL is the license used not GPL

Comment (will-moore) [reply README.md]: fixed

**Review by jburel — APPROVED** [review]

**Pull request merged**